easy-overlay.js

```diff
@@ -409,6 +410,9 @@ var easyOverlay=(function(){
 		}
 		,submit:function(ajax, url, callback, $submit, $form){
 			var self = this;
+			if (self.options && typeof self.options.beforeSubmit==='function'){
+				self.options.beforeSubmit();

```

Comment:
I think it would be worth passing a parameter here, e.g. send the submit button, the form... maybe even the ajax value in case the handler wanted to do different things based on the data sent.